metricbeat/module/elasticsearch/pending_tasks/pending_tasks.go

```diff
@@ -0,0 +1,73 @@
+package pending_tasks
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/cfgwarn"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+	"github.com/elastic/beats/metricbeat/module/elasticsearch"
+)
+
+// init registers the MetricSet with the central registry.
+// The New method will be called after the setup of the module and before starting to fetch data
+func init() {
+	mb.Registry.AddMetricSet("elasticsearch", "pending_tasks", New, hostParser)
+}
+
+var (
+	hostParser = parse.URLHostParserBuilder{
+		DefaultScheme: "http",
+		PathConfigKey: "path",
+		DefaultPath:   "_cluster/pending_tasks",
```

Comment:
The tricky part with cluster stats compared to node stats is that they should not be fetched from every node but master only. I assume for this the `IsMaster` function I?m creating here will become useful: https://github.com/elastic/beats/pull/6807/files#diff-89821f8160e055daac8a00b1f0cf846aR38